.github/workflows/ci.yml

```diff
@@ -0,0 +1,78 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
```

Comment:
super nit:

I'm pretty upset that GH has not implemented support for YAML anchors yet. 
I'm not sure if we can move out this `runs-on` and the `python-versions` to be global env vars.
```
env:
    testing_os_version: ubuntu-20.04
    testing_python_version: 3.8.2


runs-on: ${{testing-os-version}}

with:
    python-version: ${{testing-python-version}}
```

I'm not sure if something like this would be supported by GH Actions. :/

Reply:
boooooooo :(

<img width="1417" alt="Screen Shot 2021-04-19 at 5 31 04 PM" src="https://user-images.githubusercontent.com/1947534/115319603-0be7a080-a135-11eb-9743-303813c87bea.png">

Reply:
This only works for python_version, but I think it's still worth doing.

Reply:
Thanks for trying this :)